### PR TITLE
Bump version to 1.2.1

### PR DESCRIPTION
**Pull request by philippjfr**
Simple version bump along with unpin of `bokeh` and `markdown-it-py` versions.

### CODE REVIEW

**Review by lorepirri — COMMENTED** [review]
why do we skip these?
```
  skip-lints:
    - python_build_tool_in_run
    - host_section_needs_exact_pinnings
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by lorepirri — COMMENTED** [review]
@philippjfr there seem to be conflicts in dependencies which need to be addressed

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ requirements:
     - wheel
     # These are also needed at build time.
     - bleach
-    - bokeh 3.1.1
+    - bokeh 3.2.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:28]: Looks like bokeh 3.2.0 dropped python 3.8 support https://github.com/AnacondaRecipes/bokeh-feedstock/blob/22d705118e1fa3f478ea6dfa66958a5f26c5eca6/recipe/meta.yaml#L14

**Review by lorepirri — APPROVED** [review]
LGTM

**Pull request merged**